src/pages/waiting-room/cat-a-mod1/waiting-room.cat-a-mod1.page.html

```diff
@@ -0,0 +1,41 @@
+<ion-header>
+  <practice-mode-banner *ngIf="isPracticeMode"></practice-mode-banner>
```

Comment:
No practice banner needed as not practiceable page currently

Reply:
Done